### PR TITLE
Spacing between ':' and digit

### PR DESCRIPTION
**Pull request by SegFault42**
minor change fot better readability

### CODE REVIEW

File: termdown.py [hunk 1]
```diff
@@ -133,9 +133,9 @@ def format_seconds_alt(seconds, start, hide_seconds=False):
         if actual_period_value > 0:
             output += str(actual_period_value).zfill(2) + ":"
```
Comment (trehn) [inline termdown.py]: The spaces need to be added here as well.

Comment (trehn) [inline termdown.py]: The spaces need to be added here as well.

**Comment by trehn** [comment]
I didn't do this originally because it looks weird with `--no-figlet`. I think the spaces should not be added when that is set (or when termdown falls back to it because there is not enough room to display figlet letters).

**Comment by aecepoglu** [comment]
Isn't this a figlet font issue. With the 3x5 figlet font it looks perfect without any spacing.

**Comment by SegFault42** [comment]
can you share a screenshot please ?

**Comment by aecepoglu** [comment]
will this webpage suffice or do you really want to see a screenshot?
http://www.figlet.org/examples.html

**Pull request closed without merging**